Makefile

```diff
@@ -31,6 +31,9 @@ VSECM_EKS_REGISTRY_URL ?= "public.ecr.aws/h8y1n7y7"
 VSECM_NAMESPACE_SYSTEM ?= "vsecm-system"
 VSECM_NAMESPACE_SPIRE ?= "spire-system"
 VSECM_NAMESPACE_SPIRE_SERVER ?= "spire-server"
+# VSECM_NAMESPACE_SYSTEM ?= "vsecm-system-custom"
+# VSECM_NAMESPACE_SPIRE ?= "spire-system-custom"
+# VSECM_NAMESPACE_SPIRE_SERVER ?= "spire-server-custom"
```

Comment:
These commented out lines are useful to test the thing with the generated k8s manifests instead of directly using helm. — An alternative is to define the environment variables, or pass these as arguments to `make`. I wanted to keep them explicit as they double as documentation too.